scripts/Packages/Package_Gullwing__QFP_SOIC_SO/ipc_gullwing_generator.py

```diff
@@ -140,10 +141,17 @@ def deviceDimensions(device_size_data):
     def generateFootprint(self, device_params, header):
         dimensions = Gullwing.deviceDimensions(device_params)
 
-        if dimensions['has_EP'] and 'thermal_vias' in device_params:
-            self.__createFootprintVariant(device_params, header, dimensions, True)
+        if 'deleted_pins' in device_params:
+            if type(device_params['deleted_pins']) is int:
+                device_params['deleted_pins'] = [device_params['deleted_pins']]
```

Comment:
Why is this (conditional) re-formating not done for hidden_pins?

Reply:
`deleted_pin` and `hidden_pins` can be an `int` or `list` type. I need a list, so if I get an int I stick it into a list.

Reply:
my point is that you are doing this only for `deleted_pins`.
where is the conversion for `hidden_pins`?